.env

```diff
@@ -0,0 +1,2 @@
+CLOUDFLARE_ACCOUNT_ID=9ea1298a8431d6f5f9a83dacc15a288e
```

Comment:
@cf-jongsik rotate thes credentials and remove this file (and i guess add `.env` to `.gitignore`